libs/dh/shared/data-access-msw/src/lib/mocks/data/marketParticipantUserRolePermissions.json

```diff
@@ -1,37 +1,32 @@
 [
   {
-    "id": "3ec41d91-fc6d-4364-ade6-b85576a91d04",
-    "name": "Test role 1",
-    "status": "Active",
-    "eicFunction": "MeteredDataAdministrator",
-    "permissions": [1, 2, 3]
+    "id": "1",
```

Comment:
nitpick: according to `SelectablePermissionsDto`, the `id` value should be a number

Reply:
Fixed